//python -m grpc_tools.protoc -I./proto --python_out=. --grpc_python_out=. ./proto/test.proto
syntax = "proto3";
// import "google/protobuf/any.proto";
import "google/protobuf/struct.proto";


service TestService {
    rpc test (TestRequest) returns (TestReply) {}
    rpc df_info (TestRequest) returns (TestReply) {}
    rpc data (TestRequest) returns (TestReply) {}
    rpc n_rows (TestRequest) returns (TestReply) {}
    rpc max_by_col (TestRequest) returns (TestReply) {}
    rpc upload_data (TestRequest) returns (TestReply) {}
    rpc sozvon (Sozvon) returns (TestReply) {}
}

message TestRequest {

    int32 nrows = 1;
    string filename = 2;
    int32 option = 3;
    string column_name = 4;
}

message Sozvon {
    int32 name = 1;
    string familyname = 2;
}
message TestReply {
    google.protobuf.Struct message = 1;

    //string message = 1;
}